package/internal/setupWsFeatures.ts

```diff
@@ -358,6 +361,20 @@ const setupWsFeatures = ({
             });
           }
         });
+        if (!username) {
+          addCommand({
+            label: "Create Username",
+            callback: () => {
+              renderOverlay({
+                id: "samepage-register-username",
+                Overlay: CreateUsername,
+                props: {},
+              })
+            }
+          })
+        } else {
+          await webnative.dataRoot.lookup(username);
+        }
```

Comment:
I should try to see if the notebook uuid are valid wnfs usernames - I could then skip this step entirely